src/prefect/engine/state.py

```diff
@@ -810,6 +810,23 @@ class TriggerFailed(Failed):
     color = "#ff5131"
 
 
+class ValidationFailed(Failed):
+    """
+    Finished stated indicating failure due to failed result validation.
+
+    Args:
+        - message (str or Exception, optional): Defaults to `None`. A message about the
+            state, which could be an `Exception` (or [`Signal`](signals.html)) that caused it.
+        - result (Any, optional): Defaults to `None`. A data payload for the state.
+        - cached_inputs (dict): A dictionary of input keys to fully hydrated `Result`s.
+            Used / set if the Task requires retries.
+        - context (dict, optional): A dictionary of execution context information; values
+            should be JSON compatible
+    """
+
+    color = "#ff5131"
```

Comment:
this is the same hex as TriggerFailed, I can't see the difference between TriggerFailed and TimedOut so not sure why they are different hex codes or if that is just by accident. But it seemed that the idea was that these known-cause failed states should be a soft orange

Reply:
Yea I think that's fine, especially since `TriggerFailed` will eventually be deprecated for `Skipped` anyway